sage-ios/SAGE/SAGE/User.swift

```diff
@@ -27,6 +27,7 @@ class User: NSObject, NSCoding {
     enum DefaultValues: Int {
         case DefaultID = -1
         case DefaultHours = -2
+        case DefaultDirectorID = -3
```

Comment:
can you make default hours 0 so it doesnt show up as -2 for users if something goes wrong